rskj-core/src/test/java/co/rsk/peg/StateForFederatorTest.java

```diff
@@ -18,70 +18,65 @@
 
 package co.rsk.peg;
 
+import static co.rsk.peg.PegTestUtils.createHash3;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import co.rsk.bitcoinj.core.BtcTransaction;
 import co.rsk.bitcoinj.core.NetworkParameters;
 import co.rsk.crypto.Keccak256;
-import org.apache.commons.lang3.tuple.Pair;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-
+import java.util.Arrays;
 import java.util.Set;
 import java.util.SortedMap;
 import java.util.TreeMap;
+import org.ethereum.util.RLP;
+import org.ethereum.util.RLPList;
+import org.junit.jupiter.api.Test;
 
-/**
- * Created by mario on 20/04/17.
- */
 class StateForFederatorTest {
 
-    private static final String SHA3_1 = "1111111111111111111111111111111111111111111111111111111111111111";
-    private static final String SHA3_2 = "2222222222222222222222222222222222222222222222222222222222222222";
-    private static final String SHA3_3 = "3333333333333333333333333333333333333333333333333333333333333333";
-    private static final String SHA3_4 = "4444444444444444444444444444444444444444444444444444444444444444";
-
-    private static final NetworkParameters NETWORK_PARAMETERS = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+    private static final NetworkParameters NETWORK_PARAMETERS = NetworkParameters.fromID(NetworkParameters.ID_MAINNET);
 
     @Test
-    void serialize() {
-        Keccak256 hash1 = new Keccak256(SHA3_1);
-        Keccak256 hash2 = new Keccak256(SHA3_2);
-        Keccak256 hash3 = new Keccak256(SHA3_3);
-        Keccak256 hash4 = new Keccak256(SHA3_4);
+    void stateForFederator_whenSerializeAndDeserialize_shouldHaveEqualState() {
+        // Arrange
+        Keccak256 rskTxHash1 = createHash3(1);
+        Keccak256 rskTxHash2 = createHash3(2);
```

Comment:
Let's use the new method that's in RskTestUtils in favor of this one that is deprecated.

No need to change it everywhere, but at least in the new places and lines we refactor

Reply:
Updated